specification/paths/Services.json

```diff
@@ -7,79 +7,91 @@
     "description": "This endpoint retrieves services.",
     "parameters": [
       {
-        "in": "query",
         "name": "filter[package_type]",
-        "type": "string",
-        "description": "Comma separated string of package types to filter on."
+        "in": "query",
+        "description": "Comma separated string of package types to filter on.",
+        "schema": {
+          "type": "string"
+        }
```

Comment:
Most query parameters are fairly unique. Filters often describe the relationship by which to filter and are therefor not very reusable.

Reply:
They are actually quite reusable when filtering on relationships, because we have quite a few relationships with the same names. However, I wont let this stop this branch from being merged.